client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -111,4 +111,25 @@ public void openIndexAsync(OpenIndexRequest openIndexRequest, ActionListener<Ope
                 listener, Collections.emptySet(), headers);
     }
 
+    /**
+     * Closes an index using the Close Index API
+     * <p>
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-open-close.html">
+     * Close Index API on elastic.co</a>
+     */
+    public CloseIndexResponse closeIndex(CloseIndexRequest closeIndexRequest, Header... headers) throws IOException {
```

Comment:
@javanna what about removing the `*Index` suffix from the method names in the `IndicesClient` - similar to what is done in the `IndicesAdminClient`, where one simply calls `create` to trigger a create index request or `close` to trigger a close index request ? Then the user will simply call `client.indices().close()` instead of `client.indices().closeIndex()`.

Reply:
that is a very good point @olcbean. I missed this subtle difference. Let's to it in a followup PR please.